samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -129,13 +129,16 @@
   // https://help.openai.com/en/articles/4936856-what-are-tokens-and-how-to-count-them
   // - Prevent large uploads by setting a file size limit (in bytes) as suggested here:
   // https://learn.microsoft.com/en-us/aspnet/core/mvc/models/file-uploads?view=aspnetcore-6.0
+  // - Deduplication similarity threshold is used to determine if a document is similar to
+  //   another document that is already in memory.
   //
   "DocumentMemory": {
     "GlobalDocumentCollectionName": "global-documents",
     "UserDocumentCollectionNamePrefix": "user-documents-",
     "DocumentLineSplitMaxTokens": 30,
     "DocumentParagraphSplitMaxLines": 100,
-    "FileSizeLimit": 1000000
+    "FileSizeLimit": 1000000,
+    "DeduplicationSimilarityThreshold": 0.8
```

Comment:
@glahaye  - do we need to run all appsettings into the deployment logic, or do default  values work as well?

Reply:
Right now, there is nothing related to DocumentMemory in the deployment templates.  So, all the config we get is from the appsettings.json document.

These values can be overridden online through the Azure Portal too.

Reply:
So, nothing to change in templates :)